client/app/src/main/java/group01/smartcar/client/CarControl.java

```diff
@@ -19,26 +18,32 @@
     private final String SUBSCRIBE_URI = "/smartcar/control/#";
     private final String STEERING_URI = "/smartcar/control/steering";
     private final String THROTTLE_URI = "/smartcar/control/speed";
+    private final String CAMERA_URI = "/smartcar/control/camera";
+    private static final int IMAGE_WIDTH = 320;
+    private static final int IMAGE_HEIGHT = 240;
+    private ImageView mCameraView;
```

Comment:
Not really an issue, but it would be nice to keep variable naming consistent. The `m` prefix is not really needed for Java class members

Reply:
Fixed in commit https://github.com/DIT112-V21/group-01/pull/33/commits/065d0fabc39b06fd18c27196ee35bd7245e3114d